### DISCUSSION TITLE
TDesign 有奖征集品牌案例

### DISCUSSION THREAD

**Discussion by anlyyao** [Tencent/tdesign#493] [Submit & Win Prizes]
与TDesign，共生长，无论你是行业头部公司还是个人开发者，只要你用过 TDesign 提升开发效率和产品UI美感，我们都邀请你参与内外应用案例品牌logo及名称征集投稿，入选案例将展示在TDesign 新版官网品牌墙处，所对应案例提供者将获得 TDesign 官方周边礼品一份，数量有限。第一期臻选30席品牌案例，作品完成度不限，截止到6月30号新版官网上线前，以平台解释为主，欢迎大家踊跃投稿~

### 提供材料：
- 产品/案例名称
- 应用截图（不对外展示仅作审核使用）及 logo

### 案例投递邮箱：
- 方式一： 在评论区提交以上材料
- 方式二：发送材料到 tdesign@tencent.com

![GitHub](https://github.com/Tencent/tdesign/assets/51158141/067e7aef-4b58-4f5a-9f46-d515113ea588)

## 案例征集结果
<img width="808" alt="截屏2024-07-19 17 38 03" src="https://github.com/user-attachments/assets/d8004913-f6c9-4670-be97-186d9700118c">
<img width="807" alt="截屏2024-07-19 17 36 59" src="https://github.com/user-attachments/assets/4c015989-c407-43b1-8ce1-6c0b4c7868e0">

<img width="867" alt="截屏2024-07-19 17 36 07" src="https://github.com/user-attachments/assets/0815fffb-ce74-47bf-aeef-1fd44140ddc5">

**Comment by sjc-web** [comment]
作为一名资深小程序开发者，很荣幸参加TDesign 有奖征集品牌案例活动，参赛作品是一款完全基于TDesign开发的小程序项目

产品名称：骰盅神器

logo：
![logo](https://github.com/Tencent/tdesign/assets/61959566/2c3739f2-2e16-4449-b5bc-86672ef16d73)


产品截图：
![cansai1](https://github.com/Tencent/tdesign/assets/61959566/afa19fee-32d2-434a-831b-3e63dad1e64d)
![cansai2](https://github.com/Tencent/tdesign/assets/61959566/35f70342-b2ae-41b1-a305-fc7af1134677)
![cansai3](https://github.com/Tencent/tdesign/assets/61959566/c2c3bc4f-3ae0-44e0-b84b-4ec8d65f7fc9)
![cansai4](https://github.com/Tencent/tdesign/assets/61959566/25f69152-e510-4a79-9ecb-9716ca994ff3)
![cansai5](https://github.com/Tencent/tdesign/assets/61959566/a7df483c-1e50-4ec7-a2a5-e510eb0f8455)
![cansai6](https://github.com/Tencent/tdesign/assets/61959566/a416e7f9-a9d8-4e59-a205-1cabd3619315)

**Comment by MmToon** [comment]
#### 作为一名小程序萌新开发者很荣幸参加本次活动，参赛作品是一款完全基于TDesign开发的项目。

- 产品名称：**同校生活**

- 产品介绍：**这是一款我校自行研发的项目，目的是为了互相交流和学习。该项目目前仍在开发中。**

- 产品LOGO：
![校徽](https://github.com/Tencent/tdesign/assets/21327423/65c42a47-eb90-4a74-a25c-384a4c034cde)

- 产品截图：
![1717479853494](https://github.com/Tencent/tdesign/assets/21327423/8313cb88-aac0-4cdc-b942-8843d70899d1)
![1717479889512](https://github.com/Tencent/tdesign/assets/21327423/d56046d0-21b5-4e5a-917c-8fef5528f530)
![1717479931013](https://github.com/Tencent/tdesign/assets/21327423/8c00bbe3-2f15-49b7-961d-f02be43c805e)

**Comment by CrossLee** [comment]
小程序名称：Teem
功能介绍：见面社交工具，[产品故事](https://www.bug.pm/2022/04/01/%E7%A4%BE%E4%BA%A4%E5%B7%A5%E5%85%B7%E5%B0%8F%E7%A8%8B%E5%BA%8F/)
小程序码：
<img src="https://user-images.githubusercontent.com/4584905/234854597-69cc6828-8976-4b23-8c1d-fd7aa45fe965.jpg" width="30%" />

页面截图：

<img src="https://user-images.githubusercontent.com/4584905/234854754-d93b83e8-3cba-4e44-a366-350e2c82b2ba.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/234854785-f36e964d-d05d-469a-b7ab-8d3a512991f3.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/234854804-f78ea766-27a7-4769-8373-392cdf9159db.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/234854843-dc306466-6c5e-42d0-bbba-0a748a9d3db8.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/234854871-4545b280-fb6f-4621-aaf0-fdd699de7c52.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/234854912-60178b04-06e6-43de-bba2-887e4c7063b9.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/234854933-d770cc52-a571-443f-81b0-0d9a21464fe3.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/234854950-f80c8ba7-28f2-4fcc-b002-6f42a9e4776e.PNG" width="30%" />


<br/>
<br/>
<br/>
<br/>



小程序名称：小云名片
功能介绍：社交名片工具，[产品故事](https://www.bug.pm/2022/05/12/%E7%94%B5%E5%AD%90%E5%90%8D%E7%89%87/)
小程序码：
<img src="https://user-images.githubusercontent.com/4584905/216499572-f05fe7c6-f17e-4b2f-82e8-78daf1d90a86.jpg" width="30%" />

页面截图：
<img src="https://user-images.githubusercontent.com/4584905/216500447-dce95a95-4e26-4c2d-887e-86c6ba42ec63.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/216500468-e2cea5ac-8690-4f0d-848a-13c3471088e1.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/216500480-f3b5de95-e8c6-4892-bf8f-b62605338337.PNG" width="30%" />
<img src="https://user-images.githubusercontent.com/4584905/216500498-5c5ed66d-a812-4a66-a46a-8d26f6e901c9.PNG" width="30%" />

_Originally posted by @CrossLee in https://github.com/Tencent/tdesign-miniprogram/discussions/1094#discussioncomment-4857728_

**Comment by catiz** [comment]
产品名称：HomeAdmin
产品介绍：家庭中后台管理系统，产品目的是为了打造一个智能家庭系统，规划功能：药品过期提醒、电量预估、市电状态、食品过期提醒、温度湿度建议、家庭物品记录等，目前正在开发中
logo
![logo](https://github.com/Tencent/tdesign/assets/64032294/d791886a-632a-4bdd-92e7-986cbd0a0125)
产品截图
![微信截图_20240614215738](https://github.com/Tencent/tdesign/assets/64032294/f32ffc4d-d8a1-4d17-b729-c494deebc790)
![微信截图_20240614215754](https://github.com/Tencent/tdesign/assets/64032294/d8fa2756-a641-4e96-a5dc-c24a1cfaff67)
![微信截图_20240614215800](https://github.com/Tencent/tdesign/assets/64032294/f9826841-5758-461e-a983-10d34a920e6e)
![微信截图_20240614215807](https://github.com/Tencent/tdesign/assets/64032294/6b62300e-7844-42f3-895c-b259ce53ebab)

**Comment by suyu610** [comment]
1. 产品名称：金相世界

2. 产品介绍：一个材料领域的学术公益网站，旨在推动金相领域发展，共建金相图库。全站（包括C端B端），都采用TDesign框架
3. 产品LOGO：
![logo-1 26cac9ae](https://github.com/Tencent/tdesign/assets/51863403/f1355a3c-98f4-4007-910c-c002cb4729c2)
5. 页面截图：

![微信图片_20240623232505](https://github.com/Tencent/tdesign/assets/51863403/4be5a32f-78c7-4522-adee-f2a365c05945)
![微信图片_20240623232745](https://github.com/Tencent/tdesign/assets/51863403/68cd273a-25a5-4056-b472-f5444dafd4b3)
![微信图片_20240623232740](https://github.com/Tencent/tdesign/assets/51863403/65da2031-451a-496a-a306-bd45d5f53cd5)

**Comment by ElanYoung** [comment]
- 产品名称：库壁啦
- 功能描述：一款壁纸小程序~
- 产品LOGO：
![库壁啦](https://github.com/Tencent/tdesign/assets/39727435/2aa38683-4408-441d-afa1-484c83445824)
- 应用截图：
![首页](https://user-images.githubusercontent.com/39727435/235864500-810dc7a3-a581-406a-b4d4-4be47538d29a.png)
![壁纸](https://user-images.githubusercontent.com/39727435/235864836-78dfb4d2-6af1-49fb-917b-ba2b5e0a9730.png)
![分类页](https://user-images.githubusercontent.com/39727435/235863064-2649f5ac-6d9a-41dc-9558-dcd611e73593.PNG)
![壁纸列表](https://user-images.githubusercontent.com/39727435/235863101-edd38a62-aa0c-47e3-b22f-a14f9baf543e.PNG)

**Comment by kk0829** [comment]
### 产品名称

蓝星球 X

### 产品介绍

蓝星球 X 是一款专为年轻一代设计的城市探索指南应用，旨在发现并分享全球各地的美食和趣味地点。通过蓝星球 X，用户可以轻松找到隐藏在城市角落的美食小店，或是未被广泛发现的旅游景点，让每一次出行都充满惊喜。

### 产品 Logo

<img src="https://github.com/user-attachments/assets/d6194bea-fa35-42e0-83c1-d906cc1d7a01" width="250" alt="首页">

### 产品截图

<img src="https://github.com/user-attachments/assets/82005acd-ccfb-4743-8c83-b3cdc94e0f7d" width="250" alt="首页"  />
<img src="https://github.com/user-attachments/assets/f4e792df-ed4e-4cf1-b1b7-3f21bf5aa225" width="250" alt="地点列表" />
<img src="https://github.com/user-attachments/assets/8eda3b63-c4cc-4fbb-9d2c-1ca17ae40128" width="250" alt="地点主页" />
<img src="https://github.com/user-attachments/assets/be757f7a-fe2b-46bc-a7a2-a58c42e76eab" width="250" alt="发笔记" />
<img src="https://github.com/user-attachments/assets/7ad3c1f1-014f-4e27-9e5a-feab5179a217" width="250" alt="活动列表" />
<img src="https://github.com/user-attachments/assets/3b04c2e6-fa10-4708-a007-a06d1b129299" width="250" alt="活动详情" />
<img src="https://github.com/user-attachments/assets/8e91d93e-4bb6-4aad-a0bb-c9058871c4a7" width="250" alt="推荐地点" />
<img src="https://github.com/user-attachments/assets/76158091-51fa-4d26-ba2c-190e5d718387" width="250" alt="推荐活动" />
<img src="https://github.com/user-attachments/assets/46261509-4148-46cb-9608-11bc57f651bd" width="250" alt="我的收藏" />
<img src="https://github.com/user-attachments/assets/09c4f1cb-801a-432c-a766-4d1a2da2ab9e" width="250" alt="City walk 列表" />
<img src="https://github.com/user-attachments/assets/da0d0d0e-eb55-4bdf-b1a7-0b048916a1ef" width="250" alt="City walk 详情" />

**Comment by gyssoft** [comment]
1 产品名称：Mind Care

2 产品介绍：多维度、多模态心理情绪特征、多级数据建模及人工智能计算模型， 结合多级大数据和AI
分析，做出科学趋向性分析与评估。
3 产品LOGO：
![logo1](https://github.com/user-attachments/assets/7ea32d10-59d3-4a1b-9736-593503760fe1)


4页面截图：

<img width="1280" alt="微信图片_20240712160939" src="https://github.com/user-attachments/assets/b00c767b-a68a-4a6b-a546-c61197b7e254">
<img width="1280" alt="微信图片_20240712160953" src="https://github.com/user-attachments/assets/f2bb0584-c952-4beb-81fe-3eb653ecbb17">
<img width="1280" alt="微信图片_20240712161011" src="https://github.com/user-attachments/assets/8eaf4d46-fdf3-4dac-9443-a595845d3b08">
<img width="1280" alt="微信图片_20240712161043" src="https://github.com/user-attachments/assets/697bcf32-f0ae-435d-8181-e564998978c0">
<img width="1280" alt="微信图片_20240712160953" src="https://github.com/user-attachments/assets/54cb0507-ae00-47bc-97db-9719d0e7e828">
<img width="1280" alt="微信图片_20240712161052" src="https://github.com/user-attachments/assets/3d4ffb42-5ce0-491e-a684-c023833382ca">
